core/Pos/Arbitrary/Core.hs

```diff
@@ -468,7 +467,7 @@ instance Arbitrary G.GenesisCoreData where
         let chop _ [] = []
             chop n l = taken : chop n dropped
               where (taken, dropped) = splitAt n l
-        allAddrs <- fmap makePubKeyAddress <$> nonrepeating (outerLen * innerLen)
+        allAddrs <- fmap makePubKeyAddress <$> vector (outerLen * innerLen)
```

Comment:
Why is this safe? `GenesisCoreData` shouldn't have repeating addresses inside b/c of `mkGenesisCoreData` restrictions.